### ISSUE TITLE
Non localized vendor and runtime bundles

### ISSUE THREAD

**Issue by alexbbt** [privatenumber/webpack-localize-assets-plugin#16]
## Is your feature request related to a problem? Please describe.
It would be great to have the ability to create non localized bundles for vendor and runtime bundles. These bundles are not actually localized, so they could be more widely cached if we didn't create a locale bundle for them.

## Describe the solution you'd like
A config option for this would be best, as it would allow a simple filename input.

## Describe alternatives you've considered
The ability to use a functional filename allows one to pass a filename without a locale placeholder for some bundles. Once #15 is done, this would be a good solution, though it would require allowing some files to not have a `locale` place holder.

**Comment by privatenumber** [comment]
I think this might be out of scope for the plugin as it can be solved with how you configure Webpack.

For example, using [DllPlugin](https://webpack.js.org/plugins/dll-plugin/) or [externals](https://webpack.js.org/configuration/externals/) to split the build up into vendor vs application build.

The root problem is your config file is telling Webpack to create the vendor chunk in the same configuration that's telling it to localize assets. Even if you were to be able to use a function for filename, Webpack would still write the vendor asset to the same path multiple times, which is only masking the problem.
